opent2t-reference/Opent2tHelper.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+var q = require('q');
```

Comment:
Yes, we should converge these.  I'd like to do that as a separate task.

---
In reply to: [101373959](https://github.com/openT2T/opent2t-cli/pull/16#discussion_r101373959) [](ancestors = 101373959)